addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -170,18 +170,12 @@
 <template id="references_block" name="Partner References Block">
     <t t-if="any([p.website_published for p in partner.implemented_partner_ids])">
         <h3 id="references">References</h3>
-        <div t-foreach="partner.implemented_partner_ids" t-as="reference" class="media">
-          <t t-if="reference.website_published">
-            <a class="pull-left" t-attf-href="/customers/#{slug(reference)}">
-                <span t-field="reference.image_small" t-options='{"widget": "image", "class": "center-block"}'/>
-            </a>
+        <div t-foreach="partner.implemented_partner_ids" t-if="reference.website_published" t-as="reference" class="media">
+            <span t-field="reference.image_small" t-options='{"widget": "image", "class": "center-block pull-left"}'/>
```

Comment:
no need for `pull-left` css class in 12.0

Reply:
does it works the `t-if` on the same tag as `t-foreach`?

Reply:
I didn't catch any issue, tested it on 10.0. It's actually like this in 12.0:
https://github.com/odoo/odoo/blob/c13298fdacf52aedb2d2fee4db98203247c5e523/addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml#L166